types/priv_validator/sign_info.go

```diff
@@ -149,11 +149,11 @@ func (lsi *LastSignedInfo) SignVote(signer types.Signer, chainID string, vote *t
 // If the HRS are equal and the only thing changed is the timestamp, it sets the timestamp to the previous
 // value and the Signature to the LastSignedInfo.Signature.
 // Else it returns an error.
-func (lsi *LastSignedInfo) SignProposal(signer types.Signer, chainID string, proposal *types.Proposal) error {
+func (info *LastSignedInfo) SignProposal(signer types.Signer, chainID string, proposal *types.Proposal) error {
```

Comment:
what's wrong with `lsi`?

Reply:
Nothing's wrong with `lsi`, it's just that some receivers for `LastSignedInfo` were named `lsi` and others were named `info`. I just settle for `info` for all of them. I will switch to `lsi` instead.

Reply:
got it. thanks